### PR TITLE
docs(datepicker): explicitly mention i18n and warn people about pitfa…

### PR DESCRIPTION
**Pull request by mmalerba**
…lls of using NativeDateAdapter

@fxck

### CODE REVIEW

**Comment by fxck** [comment]
Looks great.

File: src/lib/datepicker/datepicker.md [hunk 1]
```diff
@@ -148,6 +155,12 @@ export class MyComponent {
 }
 ```
 
+*Please note: `MdNativeDateModule` is based off of the functionality available in JavaScript's
+native Date object, and is thus not suitable for many locales. One of the biggest shortcomings of
+the native Date object is the inability to set the parse format. We highly recommend writing a
+custom `DateAdapter` that works with the formatting/parsing library of your choice, or waiting for
+the official Moment.js module.*
```
Comment (jelbourn) [inline src/lib/datepicker/datepicker.md]: On moment: I would rephrase this to not convey that Moment is the "official" way of doing i18n, but just that we plan on making a moment adapter available as a convenience

Comment (mmalerba) [inline src/lib/datepicker/datepicker.md]: Ok, just removed the moment.js reference here since it's already mentioned in a few other places that we plan to make that
